tox.ini

```diff
@@ -23,5 +24,5 @@ commands =
 #     F403 'from ansible.module_utils.basic import *' used; unable to detect undefined names
 #     H303  No wildcard (*) import.
 # Excluding our upstream submodule, and our vendored f5 configuration script.
-ignore=F403,H303
+ignore=F403,H303,R
```

Comment:
`R` is the prefix for our checks. This is bit of a hack to avoid printing out those messages more than once per run. By default we ignore the checks we add for the case where we find all python files and run the default checks against them. Then we run flake8 a second time in serial just adding the `R` checks so that those messages are printed once and only once. It's a hack but it works well.